internal/command/fn/create.go

```diff
@@ -26,10 +26,31 @@ import (
 )
 
 type Create struct {
-	Name     string `arg:"" help:"name of the function to create"`
-	Source   string `arg:"" type:"existingdir" help:"path of the source directory"`
-	Module   string `short:"m" default:"_" help:"module of the function to create"`
-	Language string `short:"l" required:"" enum:"rust,js" help:"programming language of the function"`
+	Name     string `arg:"" help:"Name of the function to create"`
+	Source   string `arg:"" type:"existingdir" help:"Path of the source directory"`
+	Module   string `short:"m" default:"_" help:"Module of the function to create"`
+	Language string `short:"l" required:"" enum:"rust,js" help:"Programming language of the function"`
+}
+
+func (c *Create) Help() string {
+	return `
+DESCRIPTION
+
+	It builds and a uploads a function with the specified name from the 
```

Comment:
There is an extra 'a'. It builds and uploads a function. Also instead of repeating specified 2 times you could say specified name from the given source.

As for the build cmd I would shrink the sentence about the --language flag in:

The "--language" flag is required, with the following possible values: [rust, js].